### PR TITLE
Made Models

### PR DESCRIPTION
**Pull request by JakubMajewicz**
I made 10 Models for data base project

### CODE REVIEW

File: BackProject/Models/Category.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+public class Category
+{
+    public int Id { get; set; }
+    public String Name { get; set; }
+
+    public ICollection<Post> Post { get; set; }
```
Comment (OliwiaMilewska) [inline BackProject/Models/Category.cs:8]: Change ICollecton to List.

File: BackProject/Models/Post.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    public class Post
+    {
+        public int Id { get; set; }
+        public String Nick { get; set; }
+        public Category CategoryId { get; set; }
+        public String Body { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public Boolean Verified { get; set; }
+
+        public ICollection<Replie> Replie { get; set; }
```
Comment (OliwiaMilewska) [inline BackProject/Models/Post.cs:14]: Change ICollecton to List.
Replie -> Reply

File: BackProject/Models/Practice.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace BackProject.Models
+{
+    public class Practice
```
Comment (OliwiaMilewska) [inline BackProject/Models/Practice.cs:5]: Change to "Internship".

File: BackProject/Models/Professor.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+{
+
+    public class Professor
+    {
+        public int Id { get; set; }
+        public String Organisation { get; set; }
+        public String Name { get; set; }
+        public String Room { get; set; }
+        public String Email { get; set; }
+        public String Best_contact { get; set; }
```
Comment (OliwiaMilewska) [inline BackProject/Models/Professor.cs:12]: HowToContact

File: BackProject/Models/Professor.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        public String TypeOfPass { get; set; }
```
Comment (OliwiaMilewska) [inline BackProject/Models/Professor.cs:13]: TypeOfExamination

File: BackProject/Models/Professor.cs [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+        public String AvailabilityOfMaterials { get; set; }
+        public String Note { get; set; }
+
+        public ICollection<Consultation> Consultation { get; set; }
```
Comment (OliwiaMilewska) [inline BackProject/Models/Professor.cs:17]: ICollection -> List

File: BackProject/Models/Replie.cs [hunk 7]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace BackProject.Models
+{
+	public class Replie
+	{
```
Comment (OliwiaMilewska) [inline BackProject/Models/Replie.cs:6]: Reply

**Review by OliwiaMilewska — CHANGES_REQUESTED** [review]

**Review by JakubMajewicz — COMMENTED** [review]
I did changes in Models files

File: BackProject/Models/Professor.cs [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+        public String Organisation { get; set; }
+        public String Name { get; set; }
+        public String Room { get; set; }
+        public String Email { get; set; }
+        public String HowToContact { get; set; }
+        public String TypeOfExamination { get; set; }
+        public String AvailabilityOfMaterials { get; set; }
+        public String Note { get; set; }
+
+        public List<Consultation> Consultation { get; set; }
```
Comment (OliwiaMilewska) [inline BackProject/Models/Professor.cs:17]: To be honest we can make the name here in plural form:
public List<Consultation> Consultations { get; set; }

File: BackProject/Models/Post.cs [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+    public class Post
+    {
+        public int Id { get; set; }
+        public String Nick { get; set; }
+        public Category CategoryId { get; set; }
+        public String Body { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public Boolean Verified { get; set; }
+
+        public List<Reply> Reply { get; set; }
```
Comment (OliwiaMilewska) [inline BackProject/Models/Post.cs:14]: To be honest we can make the name here in plural form:
public List<Reply> Replies { get; set; }

**Review by OliwiaMilewska — CHANGES_REQUESTED** [review]

**Review by JakubMajewicz — COMMENTED** [review]
I made names in Models in plural form

**Review by OliwiaMilewska — APPROVED** [review]

**Review by AdrianStankiewicz — APPROVED** [review]

**Pull request merged**